BranchMetrics/ios-branch-deep-linking-attribution
# SDK-1797 add sdk side support for meta aem

## Reference
SDK-1797 Add SDK side support for Meta AEM

## Summary
Adds support for parsing Meta AEM values

Refactored tests to workaround BNCPreferenceHelper, it's more focused on the public APIs.

## Motivation
SDK support for Meta AEM

## Type Of Change


## Testing Instructions
Open app with test URLs in the ticket, you should see the Meta value attached to open and events
See unit tests

Note for fb URI formatted links, we'll need a bit more work. Use a generic URI or Universal link to test.









cc @BranchMetrics/saas-sdk-devs for visibility.
